readme/conflict.md

```diff
@@ -1,16 +1,30 @@
 # What is a conflict?
 
-A conflict happens when one note or one attachment is modified in two different places, and then synchronised. In that case, it is not possible to determine which version of the note or attachment you want to keep, and thus a conflict is generated.
+A conflict happens when one note or one attachment is modified in two different places, and then synchronized. In that case, it is not possible to determine which version of the note or attachment you want to keep, and thus a conflict is generated.
 
-# What happens in case of a conflict?
+# How to resolve a conflict?
 
-When Joplin detects a conflict, the local note is copied to the Conflict notebook so as to avoid any data loss. Then the remote note is downloaded. You can then inspect the notes in the Conflict notebook, compare it with your other version, and copy any change that might have been overwritten.
+When Joplin detects a conflict, it creates a _Conflict_ notebook and copies the local note to it.  Then the remote note replaces the local note. You can then inspect the notes in the Conflict notebook, compare it with your other version, and copy any change that might have been overwritten.
+
+If you are sure that the local version on one of your devices was good, you can restore the 'pre-sync' version on that device: Select the note, click on (i) Note Properties, select _Previous versions of this note_, choose the good version, click _Restore_.
+
+## If you do not remember your changes since the last failed synchronization
+1. Use a merge tool, for example www.diffchecker.com
```

Comment:
There is also a conflict resolution plugin: https://joplin.github.io/website-plugin-discovery/plugin/joplin-plugin-conflict-resolution/

Reply:
I added a link (with no further instructions since I don't know the plugin)